integrated_channels/canvas/client.py

```diff
@@ -272,9 +273,10 @@ def _get_course_id_from_integration_id(self, integration_id):
         Args:
             integration_id (string): The ID retrieved from the transmission payload.
         """
-        url = "{}/api/v1/accounts/{}/courses/".format(
+        url = "{}/api/v1/accounts/{}/courses/?search_term={}".format(
             self.enterprise_configuration.canvas_base_url,
-            self.enterprise_configuration.canvas_account_id
+            self.enterprise_configuration.canvas_account_id,
+            quote(integration_id),
```

Comment:
I think through some Moodle work I've found that requests may already escape characters in query params, but I don't think it harms anything to use it here so no biggie.

Also it's part of urllib so you could use that since you're already importing it instead of requests. 🤷 
ie `from six.moves.urllib.parse import urljoin, quote`

Reply:
ok thanks, will merge this but yeah will checkout the six version too, ty